### ISSUE TITLE
Can we use this this package for downloading a file in Excel Formal?

### ISSUE THREAD

**Issue by Rahulk4128** [react-csv/react-csv#379]
I want to use this package for allowing download in Excel format to? Does this package support this?

**Comment by EliotSlevin** [comment]
No, that's not a supported feature. You could make a pipeline where you export csv, then convert to excel, then download that.

**Comment by Rahulk4128** [comment]
can you share any example or code sandbox example? It will be a great help.
________________________________
